tsconfig.json

```diff
@@ -2,14 +2,18 @@
   "compilerOptions": {
     "outDir": "./transpiled/",
     "sourceMap": true,
-    "noImplicitAny": true,
+    "noImplicitAny": false,
```

Comment:
Has this been an issue with any dependencies?

Reply:
causes a lot of issues with the artifacts that are autogenerated